bot/exts/valentines/valentine_zodiac.py

```diff
@@ -29,13 +37,127 @@ def load_json() -> dict:
             zodiacs = load(json_data)
             return zodiacs
 
-    @commands.command(name="partnerzodiac")
-    async def counter_zodiac(self, ctx: commands.Context, zodiac_sign: str) -> None:
+    def zodiac_sign_verify(self, zodiac: str) -> discord.Embed:
+        """Gives informative zodiac embed."""
+        c_zodiac = zodiac.capitalize()
+        embed = discord.Embed()
+        embed.color = Colours.pink
+        if c_zodiac in zodiac_signs:
+            log.info("Making zodiac embed")
+            embed.title = f"__{c_zodiac}__"
+            embed.description = zodiac_fact[f"{c_zodiac}"]["About"]
+            embed.add_field(name='__Full form__', value=zodiac_fact[f"{c_zodiac}"]["full_form"], inline=False)
+            embed.add_field(name='__Motto__', value=zodiac_fact[f"{c_zodiac}"]["Motto"], inline=False)
+            embed.add_field(name='__Strengths__', value=zodiac_fact[f"{c_zodiac}"]["Strengths"], inline=False)
+            embed.add_field(name='__Weaknesses__', value=zodiac_fact[f"{c_zodiac}"]["Weaknesses"], inline=False)
+            embed.set_thumbnail(url=zodiac_fact[f"{c_zodiac}"]["url"])
+        else:
+            embed.description = "Umm you gave wrong zodiac name so i aren't able to find any :sweat_smile:"
+            log.info("Wrong Zodiac name provided")
+        return embed
+        log.info("Zodiac embed ready")
```

Comment:
Why is there a log statement after the return statement? anything after the return statement will not be executed, you should move the log above the return.